js/src/utils/urls.js

```diff
@@ -10,7 +10,8 @@ export const subpaths = {
 	editFreeListings: '/free-listings/edit',
 	editCampaign: '/campaigns/edit',
 	createCampaign: '/campaigns/create',
-	editContactInformation: '/edit-contact-information',
```

Comment:
Sorry🙏 , I forgot to list a corresponding URL reference. It needs to change as well.

https://github.com/woocommerce/google-listings-and-ads/blob/09f25d86dab2b66e2d86d5bb2795650fff777ba4/src/PluginHelper.php#L147-L149

Reply:
Ah, sorry, I got it on my paper to do list, but forgot about it.

I think, the best we can do now, given, there is no single contact information page, is to point merchants to the overall settings page.

Reply:
Or do you think we should have two separate notes now?

Reply:
> I think, the best we can do now, given, there is no single contact information page, is to point merchants to the overall settings page.

I was thinking about it in the same way.

> Or do you think we should have two separate notes now?

I think this would be adjusted later. According to Figma, after the phone verification state being available from Google end, there will be a new notification box and dedicated account issue that point to the edit phone number subpage in Settings.